perma_web/api/tests/test_link_authorization.py

```diff
@@ -95,7 +95,7 @@ def test_should_reject_logged_out_users_getting_logged_in_detail(self):
     def test_should_reject_create_to_inaccessible_folder(self):
         inaccessible_folder = self.admin_user.root_folder
         response = self.rejected_post(self.list_url, expected_status_code=400, user=self.regular_user, data=dict(self.post_data, folder=inaccessible_folder.pk))
-        self.assertIn("Folder not found.", response.content)
+        self.assertIn(b"Folder not found.", response.content)
```

Comment:
These could/should probably all be changed to something like https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.SimpleTestCase.assertContains, but I don't think it's necessary at this very moment.